src/main/java/bit/javaoop/CEO.java

```diff
@@ -2,38 +2,32 @@
 import java.util.ArrayList;
 import java.util.List;
 public class CEO {
+
     private String name;
     private List<AbstractManager> ManagerLists = new ArrayList<>();
 
     public String getName() {
         return name;
     }
 
-    public void setName(String name){
-        this.name = name;
-    }
-
-    public List<AbstractManager> getManagerLists() {
+    public List<AbstractManager> ManagerList() {
         return ManagerLists;
```

Comment:
- nazewnictwo zmiennych i metod w javie camelCase z małej literki :) poza tym getManagerList było ok, 
- na co warto zwrócić uwagę to, że przy zwracaniu referencji do naszej lsity ktoś nam ją może popsuć warto w takim getterze na listę zwrócić kopię naszej listy np. w taki sposób: `return new ArrayList<>(managerList);`